Misc/NEWS.d/next/Library/2019-01-09-23-43-08.bpo-35493.kEcRGE.rst

```diff
@@ -0,0 +1,3 @@
+Use :func:`multiprocessing.connection.wait` instead of polling each 0.2
```

Comment:
Would it be possible to describe the higher level effect: reduce the latency on pool operations. I don't recall which operation is impacted. Pool.join()? Or also Pool.terminate()?

Moreover, the change only affects process pool, not thread pool, no?

Reply:
> Moreover, the change only affects the process pool, not the thread pool, no?

 Yup, only the process pool.